src/auto/versions.txt

```diff
@@ -1,4 +1,4 @@
 All files in this directory
-were generated by gir (https://github.com/gtk-rs/gir @ d978dab)
-from gir-files (https://github.com/gtk-rs/gir-files @ 77d1f70)
+were generated by gir (https://github.com/gtk-rs/gir @ 7f5a2b5)
+from gir-files (https://github.com/gtk-rs/gir-files @ 469db10)
```

Comment:
Change in `versions.txt` but no submodule update? That why first Travis build failed.